test/lib/unit/api/api-config_spec.js

```diff
@@ -69,17 +69,31 @@ describe('ApiConfig', () => {
          });
 
          it(`will test if the GET /version route assigned by api/api-config.js is working fine`, function (done) {
-           request.get("/api-package-1-namespace/version")
+           request.get("/version")
                 .expect(200)
                 .then(res => {
-                    expect(JSON.parse(res.text).name).toBe("api-package-1");
-                    expect(JSON.parse(res.text).version).toBe('0.0.1');
+                    res.text = res.text ? JSON.parse(res.text) : {};
+                    expect((res.text[0]).api).toBe("main-package");
+                    expect((res.text[0]).version.version).toBe('0.0.1');
```

Comment:
@ankitgyawali Why is the version property a nested value? The response data does not appear to be correct.